projects/packages/jetpack-mu-wpcom/src/features/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-launchpad.php

```diff
@@ -122,7 +122,7 @@ public function get_data( $request ) {
 			'launchpad_screen'   => get_option( 'launchpad_screen' ),
 			'checklist_statuses' => get_option( 'launchpad_checklist_tasks_statuses', array() ),
 			'checklist'          => wpcom_get_launchpad_checklist_by_checklist_slug( $checklist_slug ),
-			'is_enabled'         => wpcom_launchpad_checklists()->is_launchpad_enabled(),
+			'is_enabled'         => wpcom_get_launchpad_is_enabled( $checklist_slug ),
```

Comment:
If we change this now, in order to avoid breaking production, do we need to add the `is_enabled_callback` to all the active task lists? Or is everything in production still looking directly at `launchpad_screen`?

Reply:
We won't need to add the `is_enabled_callback` to the task lists now, as `wpcom_get_launchpad_task_list_is_enabled` only uses the callback if it's defined(and returns something different from null). Otherwise, it will fall back to `wpcom_launchpad_checklists()->is_launchpad_enabled()`, which uses the `launchpad_screen` as you mentioned.

Reply:
Gotcha, the fallback was tripping me up. 😅 

This works for now, but to keep this function tight, let's plan to refactor the other task lists to use `is_enabled_callback` -- they can point directly to `wpcom_launchpad_checklists()->is_launchpad_enabled()` and remove the need for a fallback in this function. LMK if that makes sense!

Reply:
Yes, totally makes sense. I will create a new issue to tackle this refactor.

Reply:
Here we go: https://github.com/Automattic/wp-calypso/issues/77407